src/csharp/Grpc.Tools/Grpc.Tools.csproj

```diff
@@ -35,20 +35,24 @@
     <PackageId>Grpc.Tools</PackageId>
     <Authors>The gRPC Authors</Authors>
     <Copyright>Copyright 2018 The gRPC Authors</Copyright>
-    <Description>gRPC and Protocol Buffer compiler for managed C# and native C++ projects.
+    <Description>gRPC and Protocol Buffer compiler for C# projects.
```

Comment:
I checked other packages on nuget and when they provide a README.md, their descriptions is usually a single line with the very basic info - let's update the description here to match that.

![image](https://user-images.githubusercontent.com/9939684/214001123-c9075136-b758-47bc-8943-c1118d86ad17.png)